ManageIQ/integration_tests
# Control explorer (new PR)

A new PR for Control/Explorer (the old one ( #478 ) got broken so badly I decided to move it to new branch)

There will be some blank spaces in Compliance policies, but I will not do it in this PR, I'll leave it to later ones.
# How to test?

Some basic tests to verify functionality can be run:

``` bash
py.test -v -k 'test_explorer_basic'
```

Action testing is in test_explorer_actions_invocation.py and you can choose which providers you want to use

``` bash
py.test -v -k 'test_explorer_actions_invocation and (provider_id or provider_id2, ...)'
```
# Requirements?

`datastores_not_for_provision:` in the root of cfme_data.yaml. It's a list of datastores that should not be tagged for provision through SOAP. To figure it out, just look into any fully populated CFME instance and put all `iso` and `do_not_use` type datastores there. (or ask me :smiley: )
